readme.md

```diff
@@ -8,6 +8,14 @@ The machines in the labs can sometimes be quite difficult to work especially wit
 
 And then restart your shell for it to take effect.
 
+## Brew only installation
+
+This script can also be used just to install brew without zsh.
+
+```bash
+bash -c "$(curl -fsSl https://raw.githubusercontent.com/STAOJ/sta-setup/master/setup.sh)" -- -h
```

Comment:
I think you mean `-B`? Also did you test this code? Not sure if this will actually pass `-h` into the setup script. Perhaps you should try using an environment variable instead like what the homebrew install script does:
```
NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
```

Reply:
Yes, I did - thanks for catching the typo!
This worked for me on testing,  but after doing some further testing it seems to be temperamental and doesn't always work... 

(this seems to be to do with `$0` and `$BASH_SOURCE`)

I've updated this in 9ca4782, and it works better now :)